playbooks/openstack-prepare_caasp_workers_for_update.yml

```diff
@@ -0,0 +1,36 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+- hosts: caasp-admin:caasp-masters:caasp-workers
+  gather_facts: no
+  tasks:
+    - name: Ensure all the vars are loaded
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Pre-flight checks for the role
+      assert:
+        that:
+          - caasp_registration_key is defined
+
+    - name: Register node in SCC
+      command: "SUSEConnect -r {{ caasp_registration_key }}"
+
+    - name: Disable the transactional-update timer
+      command: "systemctl disable --now transactional-update.timer"
```

Comment:
should we re-enable it afterwards? Is there a reason to disable it?

Reply:
TBH I have no idea